pom.xml

```diff
@@ -73,7 +73,6 @@
     <dependency.feel.version>1.17.7</dependency.feel.version>
     <dependency.findbugs.version>3.0.2</dependency.findbugs.version>
     <dependency.guava.version>33.2.0-jre</dependency.guava.version>
-    <dependency.httpcore5.version>5.2.5</dependency.httpcore5.version>
```

Comment:
this conflicted with camunda/camunda already being on a new minor release of http core. I decided to remove it as there is no convergence issue anymore.